src/store/state/state-model.ts

```diff
@@ -79,6 +79,7 @@ export const stateModel: StateModelIF = {
   shareStructureStep: {
     valid: false,
     changed: false,
+    resolutionDates: [],
```

Comment:
Did you see that Andre created a new object in here for "new alteration data"? Would that be a better place for this (or for the new resolution dates)?

https://github.com/bcgov/business-edit-ui/blob/4bf1bd88d855370d60edcd623cf20adf85bb616f/src/store/state/state-model.ts#L21

Reply:
ahh i see. 
I'm not sure. I'll have to look at it a bit more. 
Whats the idea? an object that contains data exclusive to Alterations? 
I think we already have that with the alteration object body.

Reply:
Yes, the idea was a store object for "current alteration data" as there wasn't one already that lent itself to what André was adding. I know that other alteration data is elsewhere in the store. The hope here was to start a new thing that other devs would migrate to (instead of creating more tech debt).

Reply:
So if we mean all current alteration data, this will require some rework of actions, getters, mutations etc of all the components so far. Will also mean a new set of them separate from the current ones that are used for both filings. 
Also, the saving of drafts, parsing of drafts etc.

Not a bad idea, but a task that deserves its own ticket.

Reply:
No, change nothing now (unless you are adding data that belongs in this new object).

This is "data management" that we will also have to think about in Create UI with all the new filing types.

Yes, there's tech debt in moving other data into this new object (if it makes sense). Leave that alone for now.

Point is, we're trying to avoid further tech debt :)

Reply:
Ok sounds good. I'll give your RFC a deep dive asap, sounds like this is all related.